.rubocop.yml

```diff
@@ -0,0 +1,21 @@
+inherit_from: .rubocop_todo.yml
+
+AllCops:
+  Exclude:
+    - "vendor/**/*"
+    - "tests/integration/**/roles/*"
+    # ignore these until rubocop is incorportaed into qansible
+    - Rakefile
+    - Gemfile
+Style/StringLiterals:
+  # needs agreement
```

Comment:
are you suggesting to set the value you have mentioned? your comment often makes me wonder what the point/question/intention is.